recipe/meta.yaml

```diff
@@ -1,39 +1,45 @@
 {% set name = "pypyodbc" %}
-{% set version = "1.3.5" %}
+{% set version = "1.3.6" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.2.zip"
-  sha256: 90a9a1fdbe5c06b875155d82562beb50bb22743190ca9b46a2cb71030ee65496
+  # Use GH for LICENSE file
+  url: https://github.com/pypyodbc/pypyodbc/archive/refs/tags/v1.3.6.tar.gz
+  sha256: 14cdc66381c4204578f78512f0eb24515e0ab114f8de0cf398862a53407d99a2
 
 build:
-  number: 5
-  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
     - python
-    - unixodbc  # [unix]
+    - setuptools
+    - wheel
+    - unixodbc          # [unix]
   run:
     - python
 
 test:
   imports:
     - pypyodbc
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: "https://github.com/jiangwen365/pypyodbc"
+  home: https://github.com/pypyodbc/pypyodbc
```

Comment:
For other reviewers, [here](https://github.com/jiangwen365/pypyodbc/blob/master/README.md) is an explanation that the repo has been moved to  https://github.com/pypyodbc/pypyodbc